### DISCUSSION TITLE
🚀 Animation in Orbit should be easier to use from a developer standpoint

### DISCUSSION THREAD

**Discussion by fraincs** [gsoft-inc/sg-orbit#1110] [Ideas]
## Description

Animations values are currently exposed as tokens, this does not cover all the developers uses cases and makes it harder for them to have access to reusable animations such as a fade-in. They currently need to toggle a class to trigger animation or transition.
Components could ben produced such as Fade-in Fade-out etc using our tokens behind the scene.

## Checklist

- [ ] Documentation

**Comment by patricklafrance** [comment]
Old stuff, is it still in play?

**Comment by fraincs** [comment]
This could be moved to discussion, and take the opinion of the devs to figure how useful this would be vs the effort needed for coding and maintaining this.

**Comment by patricklafrance** [comment]
Sounds like a plan!
